web/client/reducers/maplayout.js

```diff
@@ -17,12 +28,35 @@ import assign from 'object-assign';
  *
  * @memberof reducers
  */
-function mapLayout(state = { layout: {}, boundingMapRect: {}, boundingSidebarRect: {} }, action) {
+function mapLayout(state = { layout: {}, boundingMapRect: {}, boundingSidebarRect: {}, dockPanels: { left: leftDockPanels, right: rightDockPanels } }, action) {
```

Comment:
see previous comment

```suggestion
function mapLayout(state = { layout: {}, boundingMapRect: {}, boundingSidebarRect: {}, dockPanels: { left: DEFAULT_LEFT_DOCK_PANELS, right: DEFAULT_RIGHT_DOCK_PANELS } }, action) {
```